samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
That's good to know. I don't have a use case for Core CLR and Mono at the moment. The only reason that I would support them is if we thought it useful for the sample. Most of my web development is in ASP.NET and hosted in an Azure Web App.